api/lib/domain/usecases/create-user-from-pole-emploi.js

```diff
@@ -16,8 +16,8 @@ module.exports = async function createUserFromPoleEmploi({
   const userInfo = await authenticationService.getPoleEmploiUserInfo(userCredentials.idToken);
 
   if (!userInfo.firstName || !userInfo.lastName || !userInfo.externalIdentityId) {
-    logger.error(`Un des champs obligatoires n'a pas été renvoyé par /userinfo: ${JSON.stringify(userInfo)}.`);
-    throw new InvalidExternalAPIResponseError('API PE: les informations utilisateurs récupérées sont incorrectes.');
+    logger.error(`A mandatory filed has not been returned by /userinfo: ${JSON.stringify(userInfo)}.`);
```

Comment:
typo, grammaire et précision :
```suggestion
    logger.error(`A mandatory field was not found in a Pole Emploi ID token: ${JSON.stringify(userInfo)}.`);
```

Reply:
:pray: J'ai corrigé la grammaire + orthographe
Par contre, @yahyayouness [a fait un retour](https://github.com/1024pix/pix/pull/3034#discussion_r641402012) il y a quelques jours dans le sens contraire, pour indiquer que la source est bien /userinfo, et pas l'ID token
WDYT ?

Reply:
Je veux bien mais il est où cet appel à `/userinfo` ? Je n'arrive pas à en trouver (aucune référence à `config.poleEmploi.userInfoUrl` à part pour la fixer !). En revanche `authenticationService.getPoleEmploiUserInfo(idToken)` se contente d'extraire des informations du token : https://github.com/1024pix/pix/blob/4c005a986173368004ba548f674c8835877deb3b/api/lib/domain/services/authentication-service.js#L56

Reply:
Documenté dans https://github.com/1024pix/pix/pull/3065, à  corriger ici

Reply:
Je propose de sortir ce sujet (correction anglais) de la PR 
=> à rajouter à https://github.com/1024pix/pix/pull/3268